packages/rum-core/test/performance-monitoring/capture-navigation.spec.js

```diff
@@ -118,8 +120,9 @@ describe('Capture hard navigation', function() {
         transactionEnd
       )
       expect(spans.map(mapSpan)).toEqual([
-        { name: 'Fire "DOMContentLoaded" event', _end: 1498, _start: 1464 },
```

Comment:
Is removing `Fire "DOMContentLoaded" event` intentional? I see that the difference in the event entries is zero, but I think we should provide the required entries so that these spans are tested properly. 
IMO, over time this would result in our tests becoming more comprehensive.

Reply:
Will check what I can do, it was hard to satisfy all the values.